### PR TITLE
🏴‍☠️ Frontal Gyrus Alteration Type-AR 🏴‍☠️

### PR DESCRIPTION
**Pull request by Flaborized**
## About the PR 

Adds a new speech mutation, available in genetics and through drinking grog. It's a silly pirate voice!! It's very silly and nautical, with some quite dramatic effects (that are all find-replace) such as "in" being changed to "aboard" and "door" being "hatch". 

## Why's this needed? 

Fun for admin gimmicks, silly funny addition to genetics and to grog. Pirate puns and silliness! Ahoy!

## Changelog 

```changelog
(u)Flaborized
(+)Added a new accent mutation, Frontal Gyrus Alteration Type-AR. Arrr!!🏴‍☠️
```

### CODE REVIEW

**Comment by Flaborized** [comment]
Also accepting any good funny suggested find/replace words

File: strings/language/pirate.txt [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+give@=hand over
+dude@=matey
+gun@=flintlock
+guns@=flintlocks
+hall@=poopdeck
+hallway@=poopdeck
+ha@=arg
+haha@=argarg
+hahaha@=argargarg
+hop@=first mate
```
Comment (TobleroneSwordfish) [inline strings/language/pirate.txt:90]: may be awkward if ye tries to talk about hoppin' around on yer peg leg

**Review by TobleroneSwordfish — COMMENTED** [review]
Yarrr!

**Comment by Flaborized** [comment]
whoops fixed those !

**Comment by soleilan** [comment]
I bring suggestions because I love this

```
amazing/great/etc=first rate
avoid=give a wide berth
avoided=gave a wide berth
avoiding=giving a wide berth
avoids=gives a wide berth
car=?frigate, galley? idk
deadchat?idk=dead men tell no tales (probably bad idea but funny phrase)
depressed=got the blues
drunk=bombed
greyriders=scourges of the seven seas
greytider=scourge of the seven seas
law=code of conduct
laws=codes of conduct
legitimate=above board
lol=yo-ho-ho
magnet=fishing net
negotiate/talk/speak=parley
negotiated/talked/spoke=parleyed
nonsense=bilge
omg=by neptune's beard
reward=bounty
rewarding=bountiful
rewards=bounties
scam=hornswaggle
scammed=hornswaggled
scamming=hornswagglin'
scams=hornswaggles
scavenger=privateer
scavengers=privateers
smart=savvy
stranded=marooned
syndicate=I feel like syndies should be something fun, jolly rogers men? idk
telescope=spyglass (quantum spyglass!)
theft=plunder (I want people to be arrested for grand plunder)
understood=aye aye
wallet=coffer
woah=blimey
wow=blimey
wtf=blimey
???=crow's nest (i want something to be crows nest dammit)
```

File: strings/language/pirate.txt [hunk 2]
```diff
@@ -0,0 +1,226 @@
+are@=be
+am@=be
+ahead@=bow
+front@=bow
+behind@=aft
+back@=aft
```
Comment (Tarmunora) [inline strings/language/pirate.txt:6]: I'd probably either use fore/aft or bow/stern, not bow/aft

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: strings/language/pirate.txt (hunk 2 above)
Comment (Tarmunora) [reply strings/language/pirate.txt:6]: Also could map North/East/South/West to Fore/Starboard/Aft/Port

Comment (TobleroneSwordfish) [reply strings/language/pirate.txt:6]: but NESW are absolute directions, and Fore/Starboard/Aft/Port are relative!

Comment (Flaborized) [reply strings/language/pirate.txt]: I changed the cardinal directions! and went with bow/stern. I like the idea of people trying to cheat the directional system by using n/e/s/w but still using the pirate-y speak

**Comment by Flaborized** [comment]
> I bring suggestions because I love this
> 
> ```
> amazing/great/etc=first rate
> avoid=give a wide berth
> avoided=gave a wide berth
> avoiding=giving a wide berth
> avoids=gives a wide berth
> car=?frigate, galley? idk
> deadchat?idk=dead men tell no tales (probably bad idea but funny phrase)
> depressed=got the blues
> drunk=bombed
> greytiders=scourges of the seven seas
> greytider=scourge of the seven seas
> law=code of conduct
> laws=codes of conduct
> legitimate=above board
> lol=yo-ho-ho
> magnet=fishing net
> negotiate/talk/speak=parley
> negotiated/talked/spoke=parleyed
> nonsense=bilge
> omg=by neptune's beard
> reward=bounty
> rewarding=bountiful
> rewards=bounties
> scam=hornswaggle
> scammed=hornswaggled
> scamming=hornswagglin'
> scams=hornswaggles
> scavenger=privateer
> scavengers=privateers
> smart=savvy
> stranded=marooned
> syndicate=I feel like syndies should be something fun, jolly rogers men? idk
> telescope=spyglass (quantum spyglass!)
> theft=plunder (I want people to be arrested for grand plunder)
> understood=aye aye
> wallet=coffer
> woah=blimey
> wow=blimey
> wtf=blimey
> ???=crow's nest (i want something to be crows nest dammit)
> ```
Added a few of these that I liked, thanks! 🌟

**Pull request merged**